src/decoderbufs.c

```diff
@@ -702,14 +728,18 @@ static void pg_decode_change(LogicalDecodingContext *ctx, ReorderBufferTXN *txn,
         rmsg.old_tuple =
             palloc(sizeof(Decoderbufs__DatumMessage*) * tupdesc->natts);
         rmsg.n_old_tuple -= tuple_to_tuple_msg(rmsg.old_tuple, relation,
-                           &change->data.tp.oldtuple->tuple, tupdesc);
+                           &change->data.tp.oldtuple->tuple, tupdesc, NULL);
       }
       break;
     default:
       Assert(0);
       break;
   }
 
+  if (indexT != NULL) {
```

Comment:
No behavioural difference, but easier to follow: `if (index_rel != NULL)`

Reply:
👍